crypto/x509v3/v3_addr.c

```diff
@@ -1195,7 +1203,11 @@ static int addr_validate_path_internal(X509_STORE_CTX *ctx,
     } else {
         i = 0;
         x = sk_X509_value(chain, i);
-        OPENSSL_assert(x != NULL);
+        if (!ossl_assert(x != NULL)) {
+            if (ctx != NULL)
```

Comment:
Well, commentary section above this function suggests that ctx->error should be set if you return 0. And below if doesn't return 0, it returns 1. In other words disregard above comment.